packages/frontend/src/lib/ContentDetailsLayout.svelte

```diff
@@ -8,33 +8,38 @@ const toggle = () => {
 };
 </script>
 
-<div class="grid w-full overflow-y-auto lg:grid-cols-[1fr_auto] max-lg:grid-cols-[auto]">
-  <div class="p-5 inline-grid">
-    <slot name="content" />
+<div class="flex flex-col w-full overflow-y-auto">
```

Comment:
You seems to be changing the layout, could you make it in another PR to ease the review and tests ?

Reply:
Sure, np. I'm going to show the warning on the recipe page in another PR